src/containers/tweet-container.js

```diff
@@ -39,14 +37,11 @@ export default class TweetContainer extends BaseContainer {
       win.webContents.send('load-media', mediaUrl, mediaType);
     });
 
-    ipcMain.on('imageDimensions', (event, width, height) => {
+    ipcMain.once('imageDimensions', (event, width, height) => {
       if (win != null) {
         const screenWidth  = Math.round((screen.width/2) - (width/2));
         const screenHeight = Math.round((screen.height/3) - (height/3));
-        let options = { x: screenWidth, y: screenHeight, width: width, height: height };
-        if (process.platform === 'darwin') {
-          Object.assign(options, { height: height+20 });
-        }
+        let options = { x: screenWidth, y: screenHeight, width: width, height: height+20 };
```

Comment:
Yes exactly, I tested the height and it should always add 20 to the height, else the image will be cropped. To adjust for the title on Linux and Windows, the padding is set to zero on the body and only +20 on mac where we have hidden titlebar